### ISSUE TITLE
R session crashes with arc.check_portal

### ISSUE THREAD

**Issue by bangen** [R-ArcGIS/r-bridge#44]
I want to open a feature service in RStudio.  I am signed into my active portal in ArcGIS Pro.  In the code below everything works as expected until I get to `arc.check_portal()`.  This call crashes R returning a 'R encountered a fatal error' message.  So far I've tried restarting RStudio and restarting my PC.  

Any ideas?  Thanks!

**Code:**
```   
library(arcgisbinding)
library(sf)
library(tidyverse)

sessionInfo()
arc.check_product()
arc.check_portal()
```

**Relevant session info:**
R version 4.0.2 (2020-06-22)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 16299)
other attached packages:
[1] arcgisbinding_1.0.1.241

**arc.check_product() return:**
product: ArcGIS Pro (12.6.0.24783)
license: Advanced
version: 1.0.1.241

**Comment by durphyco** [comment]
@bangen Having the same issue, running the same versions of R, the bridge, and ArcPro. Let me know if you have any luck.

**Comment by bangen** [comment]
@durphyco The issue was just resolved today (see issue #46).  I just installed the latest r-bridge release and ran `arc.check_portal()` and worked as expected.
